plugins/source/azure/client/client.go

```diff
@@ -116,9 +117,10 @@ func New(ctx context.Context, logger zerolog.Logger, s specs.Source, _ source.Op
 		return nil, fmt.Errorf("failed to unmarshal gcp spec: %w", err)
 	}
 
+	uniqueSubscriptions := funk.Uniq(spec.Subscriptions).([]string)
```

Comment:
I'd rather use [slices.Compact](https://pkg.go.dev/golang.org/x/exp/slices#Compact) along with [slices.Sort](https://pkg.go.dev/golang.org/x/exp/slices#Sort)
```suggestion
	uniqueSubscriptions := slices.Compact(slices.Sort(spec.Subscriptions))
```

Reply:
(because it's a part of Go tooling & they're bringing this package to main stdlib as well)

Reply:
Nice one, but I'm not sure we should change the order of how subscriptions are defined in the spec, as I would expected CloudQuery to sync those in order